scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source

```diff
@@ -101,22 +101,15 @@ object tup {
 
 <<< auto-tupling with class constructors
 object tup {
-  class Foo(t: (Int, String))(s: (Boolean, List[Int]))
-  new Foo(1, "foo")(true, Nil)
-}
->>>
-object tup {
-  class Foo(t: (Int, String))(s: (Boolean, List[Int]))
-  new Foo((1, "foo"))((true, Nil))
-}
-
-<<< auto-tupling with case class constructors
-object tup {
   case class Foo(t: (Int, String))(s: (Boolean, List[Int]))
-  Foo(1, "foo")(true, Nil)
+  new Foo(1, "foo")(true, Nil)
+  Foo(1, "foo")(true, Nil) // blocked by https://github.com/scalameta/scalameta/issues/846
```

Comment:
It may take awhile since we may be tempted to address https://github.com/scalameta/scalameta/issues/822 first. Currently, sugars are plain strings with no fany info like symbols. It's totally doable, just need to figure a way to position the sugar symbols.